zathura/zathura.c

```diff
@@ -651,13 +652,14 @@ document_open(zathura_t* zathura, const char* path, const char* password,
   }
 
   /* read history file */
+  char first_page_column_list_default[] = "1:2";
```

Comment:
I may have mixed `first_page_column_list` and `first_page_column_list_default`, so that might not be an issue after all.